src/lib/css/mixins/_design.scss

```diff
@@ -7,8 +7,7 @@
   }
   $elevate: 0.125rem * $strength;
 
-  box-shadow: 0 $elevate * $coeff $elevate 0
-    add-transparency(var(--shadow), calc(24% + 8% * $strength));
+  box-shadow: 0 $elevate * $coeff $elevate 0 with-alpha(var(--shadow), calc(24% + 8% * $strength));
```

Comment:
I know that in math, `*` is priorizited over `+`, but let's please use braces here. Even if this is just a formatting change. ` calc(24% + (8% * $strength))`

Reply:
Sure